relationships/synthesis/APM_CALLS_DATABASE.yml

```diff
@@ -0,0 +1,20 @@
+relationships:
+  - version: 1
+    name: apmCallsDatabase
+    origins:
+      - APM Metrics
+    conditions:
+      - attribute: metricName
+        regex: "^datastore/instance/[^/]*/([^/]*\\.rds\\.amazonaws\\.com[^/]*)/.*"
+    relationship:
+      expires: P75M
+      relationshipType: CALLS
+      source:
+        extractGuid:
+          attribute: entity.guid
+      target:
+        candidateCategory: DATABASE
+        fields:
+          - field: endpoint
+            attribute: metricName__4
```

Comment:
Is it the correct one still? Didn't it changed? cc @EmiliaRipeanu @naxhh

Reply:
this one looks correct to me